src/views/v2/commerce/checkout/cart/item/details/toggle.php

```diff
@@ -28,20 +26,25 @@
 <div class="tribe-tickets__commerce-checkout-cart-item-details-toggle">
 	<button
 		type="button"
-		class="tribe-common-b3 tribe-tickets__commerce-checkout-cart-item-details-button--more"
-		aria-controls="<?php echo esc_attr( $aria_controls ); ?>"
-		tabindex="0"
-	>
-		<span class="screen-reader-text tribe-common-a11y-visual-hide"><?php esc_html_e( 'Open the ticket description in checkout.', 'event-tickets' ); ?></span>
-		<?php echo esc_html_x( 'More info', 'Opens the ticket description', 'event-tickets' ); ?>
+		class="tribe-common-b2 tribe-common-b3--min-medium tribe-tickets__commerce-checkout-cart-item-details-button--more"
+		aria-controls="<?php echo esc_attr( $aria_controls ); ?>"	>
+		<span class="screen-reader-text tribe-common-a11y-visual-hide">
+			<?php esc_html_e( 'Open the ticket description in checkout.', 'event-tickets' ); ?>
+		</span>
+		<span class="tribe-tickets__commerce-checkout-cart-item-details-button-text">
+			<?php echo esc_html_x( 'More info', 'Opens the ticket description', 'event-tickets' ); ?>
+		</span>
 	</button>
 	<button
 		type="button"
-		class="tribe-common-b3 tribe-tickets__commerce-checkout-cart-item-details-button--less"
+		class="tribe-common-b2 tribe-common-b3--min-medium tribe-tickets__commerce-checkout-cart-item-details-button--less"
 		aria-controls="<?php echo esc_attr( $aria_controls ); ?>"
-		tabindex="0"
 	>
-		<span class="screen-reader-text tribe-common-a11y-visual-hide"><?php esc_html_e( 'Close the ticket description in checkout.', 'event-tickets' ); ?></span>
-		<?php echo esc_html_x( 'Less info', 'Closes the ticket description', 'event-tickets' ); ?>
+		<span class="screen-reader-text tribe-common-a11y-visual-hide">
+			<?php esc_html_e( 'Close the ticket description in checkout.', 'event-tickets' ); ?>
+		</span>
+		<span class="tribe-tickets__commerce-checkout-cart-item-details-button-text">
+			<?php echo esc_html_x( 'Less info', 'Closes the ticket description', 'event-tickets' ); ?>
+		</span>
 	</button>
```

Comment:
Curious what the plan is for this, are we still going to be using 2 buttons? Or will we have 1 button and handle changing text with JS?

Reply:
Yes, as mentioned in the PR description. We're keeping two buttons to be consistent [with what we have right now on the other place](https://github.com/the-events-calendar/event-tickets/blob/master/src/views/v2/tickets/item/content/description-toggle.php). We can always revisit this in the future.